src/containers/Group/Orders/Order/Edit/Products/Item/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import { getProductName } from 'presenters/product';
+import { edit, create, destroy } from 'redux/modules/order_lines';
+
+import QuantityEditForm from 'components/forms/order/QuantityEdit';
+
+class Item extends Component {
+  static propTypes = {
+    product: PropTypes.object.isRequired,
+    order: PropTypes.object.isRequired,
+    editOrderLine: PropTypes.func.isRequired,
+    createOrderLine: PropTypes.func.isRequired,
+    destroyOrderLine: PropTypes.func.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.handleSubmit = this._handleSubmit.bind(this);
+  }
+
+  _handleSubmit(fields) {
+    const { order, editOrderLine, createOrderLine, destroyOrderLine } = this.props;
+    const { productId, orderLineId } = fields;
+    const quantity = fields[`quantity_${productId}`];
+
+    if (!orderLineId) {
+      return createOrderLine({ order_id: order.id, product_id: productId, quantity });
+    }
+
+    if (quantity === '0') {
```

Comment:
The problem here is that the input field in the form returns strings, not numbers. But I still need to investigate how to manage with number related input forms 😬